### PR TITLE
Set up CI pipeline

### PR DESCRIPTION
**Pull request by benkrikler**
Adds travis-CI integration and the necessary configs to run the tests and coverage

### CODE REVIEW

**Comment by benkrikler** [comment]
Ok, I think this is complete and ready to merge.  I'll leave it for now for @eduardo-rodrigues and / or @mayou36 to take a look.  Once this is in, I think we should be able to tag things here and have them auto-deployed to pypi, addressing #8.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+language: python
+
+matrix:
+  include:
+     - python: 2.7
+     - python: 3.6
```
Comment (mayou36) [inline .travis.yml]: Maybe test against 3.7? As it's specified in the setup.py as supported.

If 3.7 _and_ 2.7 works, all in between should as well. If we have 3.6, probably 3.7 breaks (new keywords etc.).

Or just add all supported Python versions?

**Comment by eduardo-rodrigues** [comment]
Looks good to me. Feel free to merge.

One could indeed test on 2.7 and 3.7 on Travis, but it won't make a big difference for a data package with so little code.

File: .travis.yml (hunk 1 above)
Comment (benkrikler) [reply .travis.yml]: Very good point, and trivial to solve.  I've added python 3.7 to the matrix, and swapped it to be the version we deploy from.

Comment (benkrikler) [reply .travis.yml]: Wait, no, I take back the comment about it being trivial to add, seems you need to give the CI pipeline sudo permissions or take a completely different approach (https://github.com/travis-ci/travis-ci/issues/9815, https://github.com/travis-ci/travis-ci/issues/9069#issuecomment-425720905).  Are we ok with me reverting the 3.7 change and leaving this just testing 3.6 for now?

Comment (mayou36) [reply .travis.yml]: True, let's keep it simple and just use 3.6.

**Comment by benkrikler** [comment]
Excellent, thanks both @mayou36 and @eduardo-rodrigues.  I'm going to merge this in now.  And then we'll see if I've not made a mess of the auto-deployment stuff...

File: .travis.yml (hunk 1 above)
Comment (eduardo-rodrigues) [reply .travis.yml]: Yeah, that's not trivial and one of the reasons why I set up Azure pipelines for the particle package. Works nicely and tests the code with Py3.7 and most used platforms :-).

**Pull request merged**